R/corona.R

```diff
@@ -108,7 +108,7 @@ server <- function(id) {
             name = NULL,
             breaks = breaks_pretty(8),
             date_minor_breaks = "1 days",
-            date_labels = "%-d.%-m.",
```

Comment:
Date formatting seems to be platform specific. Using the dashes it renders the string "-d.-m." instead of the date on my machine, while using "%d.%m." the date still has leading zeros. 
Maybe have the date format determined application wide based on the current OS?

Reply:
Maybe `%0d` works cross-platform?

Reply:
`%0d` enforces trailing zeros, which is the same behavior as `%d`.

According to this [Stackoverflow post](https://stackoverflow.com/a/62159914/2796524), `%-d` works on Unix and `%#d` on Windows. Hence, an application wide date formatting based on the current OS would be the solution.

For now, I'll change it back to `%-d`, as the production server is Unix and I want this to be merged :smile: . And I'll open an issue to fix this.